src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/UpdateSentenceDateService.kt

```diff
@@ -91,21 +93,20 @@ class UpdateSentenceDateService(
 
     if (sentenceChanges.isMaterial) {
       val isNotApprovedForHdc = !hdcService.isApprovedForHdc(
-        updatedLicenceEntity.bookingId!!,
+        licenceEntity.bookingId!!,
         sentenceDates.homeDetentionCurfewEligibilityDate,
       )
-      notifyComOfUpdate(updatedLicenceEntity, licenceEntity, licenceId, dateChanges, isNotApprovedForHdc)
+      notifyComOfUpdate(licenceEntity, licenceId, dateChanges, isNotApprovedForHdc)
     }
   }
 
   private fun notifyComOfUpdate(
-    updatedLicenceEntity: Licence,
```

Comment:
this didn't need to be passed in before - all the referred to fields won't have changed as a result of this update